pkg/query-service/model/v3/v3.go

```diff
@@ -267,9 +286,10 @@ func (a AttributeKey) Validate() error {
 }
 
 type FilterAttributeValueResponse struct {
-	StringAttributeValues []string      `json:"stringAttributeValues"`
-	NumberAttributeValues []interface{} `json:"numberAttributeValues"`
-	BoolAttributeValues   []bool        `json:"boolAttributeValues"`
+	StringAttributeValues  []string      `json:"stringAttributeValues"`
+	Int64AttributeValues   []interface{} `json:"int64AttributeValues"`
```

Comment:
Can't we use numberAttributeValues for both int64 and float64?

Reply:
If we are building queries using the type then it won't be possible in logs.

Reply:
made the change